pkg/extprom/http/instrument_server.go

```diff
@@ -72,7 +72,7 @@ func NewServerInstrumentor() ServerInstrumentor {
 			[]string{"code", "handler", "method"},
 		),
 	}
-	prometheus.MustRegister(ins.requestDuration, ins.requestSize, ins.requestsTotal, ins.responseSize)
+	reg.MustRegister(ins.requestDuration, ins.requestSize, ins.requestsTotal, ins.responseSize)
```

Comment:
this looks like its gonna panic with multiple registrations

Reply:
That's what I've thought as well. I kept the existing behaviour from prometheus handler, I assumed if it worked, this should be fine. I will send the PR upstream and then I'll have another look at it.